src/react/components/pages/train/trainPage.tsx

```diff
@@ -99,6 +102,7 @@ export default class TrainPage extends React.Component<ITrainPageProps, ITrainPa
             modelName: "",
             modelUrl: "",
             currModelId: "",
+            isOnPrem: false,
```

Comment:
why is there a state added here..? I don't see it being used anywhere and I don't think it's needed (state is for re-rendering purposes)